### PR TITLE
Add more GSoC project ideas

### PR DESCRIPTION
**Pull request by jpouellet**

### CODE REVIEW

File: pages/GSoC.md [hunk 1]
```diff
@@ -341,6 +341,17 @@ should be opened in DispVM or locally)
 
 **Mentor**:
 
+### New Qubes Manager
+**Project**: New Qubes Manager
```
Comment (marmarek) [inline pages/GSoC.md]: This have some progress already:
 - https://github.com/QubesOS/qubes-issues/issues/2132#issuecomment-255106656
 - https://github.com/kalkin/manager-new
 - https://github.com/kalkin/qubes-dbus

Comment (jpouellet) [inline pages/GSoC.md]: I was not aware the work was ongoing. What do you suggest?
- Include those links in project description?
- Remove project proposal as it's already being handled?

Comment (marmarek) [inline pages/GSoC.md]: I'd remove this one, as (if everything goes according to plan...) at the time of GSoC, it should be already completed.

**Comment by jpouellet** [comment]
Reverted qubes-manager proposal commit as per @marmarek's suggestion & added one about reproducible builds.

**Pull request merged**